.github/workflows/ci-core.yml

```diff
@@ -449,3 +470,47 @@ jobs:
       - run: make gomodtidy
       - name: Ensure clean after tidy
         run: git diff --minimal --exit-code
+
+  run-frequency:
+    name: Scheduled Run Frequency
+    outputs:
+      one-per-day-frequency: ${{ steps.check-time.outputs.one-per-day-frequency || 'false' }}
+      two-per-day-frequency: ${{ steps.check-time.outputs.two-per-day-frequency || 'false' }}
+      four-per-day-frequency: ${{ steps.check-time.outputs.four-per-day-frequency || 'false' }}
+      six-per-day-frequency: ${{ steps.check-time.outputs.six-per-day-frequency || 'false' }}
```

Comment:
i don't think we are using this?

Reply:
We are using one, two, and four per day frequency across CI Core.